s3/src/main/scala/akka/stream/alpakka/s3/impl/HttpRequests.scala

```diff
@@ -43,11 +53,15 @@ private[alpakka] object HttpRequests {
       implicit ec: ExecutionContext,
       conf: S3Settings): Future[HttpRequest] = {
 
+    //Do not let the start PartNumber,ETag and the end PartNumber,ETag be on different lines
+    //  They tend to get split when this file is formatted by IntelliJ
     val payload = <CompleteMultipartUpload>
-                    {
-                      parts.map { case (partNumber, etag) => <Part><PartNumber>{ partNumber }</PartNumber><ETag>{ etag }</ETag></Part> }
-                    }
-                  </CompleteMultipartUpload>
+      {parts.map { case (partNumber, etag) => <Part>
```

Comment:
:) Yes, in fact this has caused me a lot of grief! Still not the best solution as IntelliJ will just wreck this file.  I have reverted this change as well as implemented a more permanent solution but is specific to IntelliJ in the next commit.